charts/portworx/templates/portworx-controller.yaml

```diff
@@ -114,8 +116,17 @@ spec:
           httpGet:
             host: 127.0.0.1
             path: /healthz
+            {{- if semverCompare ">=1.22" .Capabilities.KubeVersion.GitVersion }}
+            {{- if $AKSorEKSInstall }}
```

Comment:
This needs to be done only for AKS, and not for EKS.

Reply:
updated